packages/wonder-blocks-grid/components/flex-cell.js

```diff
@@ -9,11 +9,11 @@ import type {GridSize} from "../util/types.js";
 
 type Props = {
     /** Should this cell be shown on a Small Grid? */
-    small?: boolean,
+    small: boolean,
     /** Should this cell be shown on a Medium Grid? */
-    medium?: boolean,
+    medium: boolean,
     /** Should this cell be shown on a Large Grid? */
-    large?: boolean,
+    large: boolean,
```

Comment:
Why are these mandatory now?

Reply:
I see, they should've been in the first place since you're specifying default values.  Cool.  I'm usually lazy when it comes to bools and don't bother b/c `undefined` is falsy.

Reply:
I'm glad to see that react-docgen handles this case correctly.

Reply:
Yep, exactly!